cmd/todo/delete/delete.go

```diff
@@ -14,9 +14,21 @@ func New() *cobra.Command {
 		Run:   run,
 	}
 
+	cmd.Flags().IntP("number", "n", 0, "number on ordered list to delete")
+	cmd.MarkFlagRequired("number")
+
 	return cmd
 }
 
 func run(cmd *cobra.Command, args []string) {
-	log.Println("Deleting a todo doc...")
+	n, _ := cmd.Flags().GetInt("number")
+
+	files := utils.GetAllTodos()
+
+	if n == 0 || n > len(files) {
+		return
```

Comment:
Let's print something to the terminal here to let them know the number isn't vaild.